app.js

```diff
@@ -18,7 +20,10 @@ let key;
 const keyPath = `${appPath}/key.txt`;
 try { key = fs.readFileSync(keyPath) } catch (_) {}
 
-const archive = drive.createArchive(key, { live: true });
+const archive = drive.createArchive(key, {
+  live: true,
+  file: name => raf(`${appPath}/files/${name}`)
```

Comment:
i think we should put them in the downloads folder (whatever the equivalent is on each os). this points to the electron folder right?